crates/common/types/requests.rs

```diff
@@ -17,6 +20,11 @@ const DEPOSIT_TYPE: u8 = 0x00;
 const WITHDRAWAL_TYPE: u8 = 0x01;
 const CONSOLIDATION_TYPE: u8 = 0x02;
 
+lazy_static! {
+    static ref DEPOSIT_TOPIC: H256 =
+        H256::from_str("649bbc62d0e31342afea4e5cd82d4049e7e1ee912fc0889aa790803be39038c5").unwrap();
```

Comment:
Do we want to have these constants in a centralized place like https://github.com/lambdaclass/ethrex/blob/main/crates/common/constants.rs ?

Reply:
It is something specific to requests so I don't think we will be using it anywhere else to warrant it being centralized